contact-profile/contact-profile-project/.hsignore

```diff
@@ -0,0 +1,5 @@
+package.json
```

Comment:
we shouldn't include this in `.hsignore`s since if people copy the project they need to remove this before any third party deps work, been meaning to PR this elsewhere in examples as well